modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/integration.dm

```diff
@@ -122,5 +122,5 @@
 // prevents players being trapped in their brain, alive, yet limbless and voiceless
 /obj/item/bodypart/head/drop_organs(mob/user, violent_removal)
 	var/obj/item/organ/internal/brain/brain = locate(/obj/item/organ/internal/brain) in src
-	brain?.brainmob.death()
+	brain?.brainmob?.death()
 	. = ..()
```

Comment:
This should probably be signal bound to the organ remove as brains can exist in chests as well, and should be compatible with all limbs

Reply:
aeugh you are correct

Reply:
Actually, this is just ONLY a case for the head, so the HEAD can talk, it's not revived unless it is within a head container

Reply:
/obj/item/bodypart/head is specifically checked before reviving them as a head, a talking chest doesn't make much sense anyway

Reply:
Either way should probably be signalbound to the brain remove proc with a check for the brain's var/zone